org.adempiere.base/src/org/adempiere/model/GenericPO.java

```diff
@@ -132,6 +146,604 @@ protected int get_AccessLevel() {
 		return Integer.parseInt(p_info.getAccessLevel());
 	}
 
+
+
+	/* METHODS TO USE GenericPO in Documents */
+
+	/**
+	 * Verify if a column exists
+	 * @param columnName
+	 * @param throwException - must throw an exception when the column doesn't exist
+	 * @return
+	 */
+	private boolean columnExists(String columnName, boolean throwException) {
+		int idx = get_ColumnIndex(columnName);
+		if (idx < 0 && throwException)
+			throw new AdempiereException("Column " + get_TableName() +"." + columnName + " not found");
+		return (idx >= 0);
+	}
+
+	/**
+	 * Verify if a column exists
+	 * @param columnName
+	 * @return boolean
+	 */
+	private boolean columnExists(String columnName) {
+		return columnExists(columnName, false);
+	}
+

```

Comment:
it seems useful to have this 2 columnexists method as public method at PO.

Reply:
done